docs/sink/flink-connector-oceanbase-directload.md

```diff
@@ -0,0 +1,427 @@
+        statement.begin();
+        ObDirectLoadStatementExecutionId statementExecutionId =
+                statement.getExecutionId();
+        byte[] executionIdBytes = statementExecutionId.encode();
+        // Convert the execution id in byte[] form to string form so that it can be passed to the Flink-SQL job as a parameter.
+        String executionId = java.util.Base64.getEncoder().encodeToString(executionIdBytes);
+        System.out.println(executionId);
+
+        // 2. After obtaining the executionId, submit the Flink SQL job.
+
+        // 3. Enter the id of the Flink job submitted in the second step on the command line and wait for the Flink job to be completed.
+        Scanner scanner = new Scanner((System.in));
+        String flinkJobId = scanner.nextLine();
+
+        while (true) {
+            // Loop to check the running status of Flink jobs, see: https://nightlies.apache.org/flink/flink-docs-master/zh/docs/ops/rest_api/
+            JSONObject jsonObject = JSON.parseObject(new URL("http://localhost:8081/jobs/" + flinkJobId));
+            String status = jsonObject.getString("state");
+            if ("FINISHED".equals(status)) {
+                break;
+            }
+            Thread.sleep(3_000);
+        }
+
+        // 4. After waiting for the Flink job execution to FINISHED, perform the final submission action of the direct-load task.
+        statement.commit();
+
+        statement.close();
+        connection.close();
+    }
+}
+```
+
+Once executed, the records should have been written to OceanBase.
+
+## Configuration
+
+<div class="highlight">
+    <table class="colwidths-auto docutils">
+        <thead>
+            <tr>
+                <th class="text-left" style="width: 10%">Option</th>
+                <th class="text-left" style="width: 8%">Required by Table API</th>
+                <th class="text-left" style="width: 7%">Required by DataStream</th>
+                <th class="text-left" style="width: 10%">Default</th>
+                <th class="text-left" style="width: 15%">Type</th>
+                <th class="text-left" style="width: 50%">Description</th>
+            </tr>
+        </thead>
+        <tbody>
+            <tr>
+                <td>host</td>
+                <td>Yes</td>
+                <td>Yes</td>
+                <td style="word-wrap: break-word;"></td>
+                <td>String</td>
+                <td>Hostname used in direct-load.</td>
+            </tr>
+            <tr>
+                <td>port</td>
+                <td>Yes</td>
+                <td>Yes</td>
+                <td style="word-wrap: break-word;"></td>
+                <td>Integer</td>
+                <td>Rpc port number used in direct-load.</td>
+            </tr>
+            <tr>
+                <td>username</td>
+                <td>Yes</td>
+                <td>Yes</td>
+                <td style="word-wrap: break-word;"></td>
+                <td>String</td>
+                <td>The username.</td>
```

Comment:
It's better to note users what kind of username is required. For now we have two versions:
1. Pure user name, the name of the database user, like 'root'.
2. Connection user name, which can be used to create the connection by client, like 'root@test' when directly connected to observer or 'root@test#obcluster' when connected to obproxy.
3. Full user name, like the username used to connect to obproxy, in format 'root@test#obcluster' or 'obcluster:test:root'.

Reply:
Change to: The name of the database user, like 'root'. NOTE: Not the connection username like 'root@sys'.